### PR TITLE
Drop python 3.8

### PR DESCRIPTION
**Pull request by echarlaix**

### CODE REVIEW

File: .github/workflows/dev_test_optimum_common.yml [hunk 1]
```diff
@@ -12,10 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version:
-        - 3.7
-        - 3.8
-        - 3.9
+        python-version: ['3.9', '3.11']
```
Comment (michaelbenayoun) [inline .github/workflows/dev_test_optimum_common.yml:15]: Why not 3.10?

File: setup.py [hunk 2]
```diff
@@ -123,9 +123,9 @@
         "Intended Audience :: Education",
         "Intended Audience :: Science/Research",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```
Comment (michaelbenayoun) [inline setup.py:128]: Why not 3.11?

File: .github/workflows/dev_test_optimum_common.yml (hunk 1 above)
Comment (echarlaix) [reply .github/workflows/dev_test_optimum_common.yml:15]: 3.11 also has the security status : https://devguide.python.org/versions/ so should be tested

File: setup.py (hunk 2 above)
Comment (echarlaix) [reply setup.py:128]: added in https://github.com/huggingface/optimum/pull/2086/commits/44059cc1fa785ff96fbc01be0be00d64fc55e136

**Review by michaelbenayoun — APPROVED** [review]
LGTM thanks!

**Comment by echarlaix** [comment]
also adding every subpackages maintainer to make sure this works for everyone before we do any release @regisss @IlyasMoutawwakil @JingyaHuang @mht-sharma @

**Pull request merged**